department-of-veterans-affairs/va.gov-cms
# VACMS 4133: Update help text for link-to-file-or-video

## Description

See #4133. 

## Testing done
Manual testing.  This is a straight config read/write, not handled or altered in code.

## Screenshots
<img width="988" alt="Screen Shot 2021-02-03 at 10 39 24 AM" src="https://user-images.githubusercontent.com/1318579/106773054-ab08fb80-660e-11eb-9df2-bba76bc1b85b.png">
<img width="1105" alt="Screen Shot 2021-02-03 at 10 41 17 AM" src="https://user-images.githubusercontent.com/1318579/106773057-aba19200-660e-11eb-9f56-3147169c6291.png">

## QA steps
As a user with permission to edit CMS Help Page content:
1. Add a main content block
2. Verify the ACs from #4133 
    - [ ] paragraph title, thumbnail and description are updated per spec above
    - [ ] linked text field description in the paragraph is updated per spec bove

Note that while there's an additional description removed from the config, the relevant instruction re: file types is still present in the UI.  That description was apparently cruft.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
